event-tickets.php

```diff
@@ -2,7 +2,7 @@
 /*
 Plugin Name: Event Tickets
 Description: Event Tickets allows your guests to RSVP from any post, page, or event.
-Version: 4.5.0.2
+Version: 4.7dev1
```

Comment:
probably unimportant but just noting that the version in `package.json` is `4.7.0dev1`, whereas it's just `4.7dev1` here, in `src/Tribe/Main.php`, and in `readme.txt`. could be intentional but just noting it in case it's not.

Reply:
That's standard for us across the board. `package.json` needs 2 dots but we don't do that elsewhere.